refactor: move to common OCI

hanthor opened the pull request:
this is to miror https://github.com/ublue-os/bluefin/pull/3702

will eventually cover #855

hanthor committed:
refactor: move to common OCI

castrojo commented on build_scripts/20-packages.sh:
```diff
@@ -52,19 +52,10 @@ dnf -y --enablerepo copr:copr.fedorainfracloud.org:ublue-os:packages swap \
 	-x bluefin-logos \
 	-x bluefin-readymade-config \
-	ublue-os-just \
-	ublue-os-luks \
-	ublue-os-signing \
-	ublue-os-udev-rules \
 	ublue-os-update-services \
-	ublue-{motd,fastfetch,bling,rebase-helper,setup-services,polkit-rules,brew} \
-	uupd \
```
We only have the justfiles, all the other stuff you're still gonna need.

hanthor replied:
I'll add them in then

hanthor committed:
Add ublue-os packages to installation script

hanthor committed:
Remove duplicate package entry and add comments

Removed duplicate entry of 'ublue-os-update-services' from the package list and added comments regarding the upstream bug.

hanthor committed:
Modify services and enable brew-setup

Added command to disable module for uud service and enabled brew-setup service.

hanthor committed:
Enable uupd.timer and ublue services

Added services and timers for uupd and ublue setup.

hanthor committed:
Update .gitignore to ignore .zed directory

Remove specific common_dump entry and ignore the entire .zed directory.

hanthor committed:
feat: Add initial MOTD files for Bluefin LTS, including a template, general tips, and Bluefin-specific information.

hanthor committed:
Update build.sh

hanthor committed:
Update build.sh

hanthor committed:
Update Containerfile
